app/Actions/Jetstream/AddTeamMember.php

```diff
@@ -19,7 +19,7 @@ class AddTeamMember implements AddsTeamMembers
      * @param  mixed  $team
      * @return void
      */
-    public function add($user, $team, string $email, string $role = null)
+    public function add($user, $team, string $email, ?string $role = null)
```

Comment:
is the '?' after string mark added intentionally?

Reply:
Yes. In this case, this is not obvious but it indicates that the role is optional and adding more parameters in future doesn't require re-ordering of this parameter, if the new one is required.